portfolio/src/main/java/com/google/sps/servlets/CommentServlet.java

```diff
@@ -31,20 +32,20 @@ public void doPost(HttpServletRequest request, HttpServletResponse response) thr
     UserService userService = UserServiceFactory.getUserService();
     PrintWriter out = response.getWriter();
 
-    final String USERNAME_FIELD = getUsername(userService.getCurrentUser().getUserId());
-    final String COMMENT_TEXT_FIELD = request.getParameter("comment");
-    final long TIMESTAMP_FIELD = System.currentTimeMillis();
+    final String usernameField = getUsername(userService.getCurrentUser().getUserId());
+    final String commentTextField = request.getParameter(COMMENT_TEXT_KEY);
+    final long timestampField = System.currentTimeMillis();
 
     if (!userService.isUserLoggedIn()) {
       out.print("<p>Please login before posting a comment </p>");
       response.sendRedirect("/login");
       return;
     }
 
-    Entity commentEntity = new Entity("Comment");
-    commentEntity.setProperty("username", USERNAME_FIELD);
-    commentEntity.setProperty("commentText", COMMENT_TEXT_FIELD);
-    commentEntity.setProperty("timestamp", TIMESTAMP_FIELD);
+    Entity commentEntity = new Entity(COMMENT_KEY);
+    commentEntity.setProperty(USERNAME_KEY, usernameField);
+    commentEntity.setProperty(COMMENT_TEXT_KEY, commentTextField);
+    commentEntity.setProperty("timestamp", timestampField);
```

Comment:
I'd create a constant TIMESTAMP_KEY as well, for consistency =)

Reply:
Created da key